Sources/Sentry/SentryUserFeedback.m

```diff
@@ -2,7 +2,26 @@
 #import "SentrySwift.h"
 #import <Foundation/Foundation.h>
 
-@implementation SentryUserFeedback
+typedef enum : NSUInteger {
```

Comment:
`m`: I'm a bit confused as to why we need this as we have the Swift class SentryFeedback, or do you plan on removing this?

Reply:
Ah, this was an earlier attempt to try to reuse the existing class. I've switched to just duplicating everything so the old stuff can be removed after the deprecation period.